IntegrationTests/PurchaseTester/PurchaseTester/CatsViewController.swift

```diff
@@ -70,20 +70,22 @@ class CatsViewController: UIViewController {
     }
 
     @objc func manageSubButtonTapped() {
-        Purchases.shared.showManageSubscriptionModal { error in
-            print("Error opening Manage Sub Modal")
+        Purchases.shared.showManageSubscriptionModal { maybeError in
+            if let error = maybeError {
```

Comment:
Is `maybe` our convention? IMO it's unnecessary since you can shadow the original value, and the compiler is doing the work checking the types, so I think it's unnecessary noise.

Reply:
yeah it's a convention in our style guide for using internally: https://github.com/RevenueCat/purchases-ios/blob/main/SwiftStyleGuide.swift#L55

i don't have a strong opinion either way...

Reply:
it's a convention for all optional types, to more easily convey their nullability at first glance in cases like this, where the type definition isn't visible and you'd have to go into the method definition to see whether it was an `Error` or `Error?`. 
It's meant as a visual reminder for other devs to handle the `nil` case if needed